lib/trino-filesystem/src/main/java/io/trino/filesystem/memory/MemoryFileSystem.java

```diff
@@ -42,7 +42,7 @@ public class MemoryFileSystem
 {
     private final ConcurrentMap<String, MemoryBlob> blobs = new ConcurrentHashMap<>();
 
-    boolean isEmpty()
+    public boolean isEmpty()
```

Comment:
revert, if it's used in the test, package private should be enough, otherwise add `@VisibleForTest`